### ISSUE TITLE
QFAPI illegally accesses Quilt Base API

### ISSUE THREAD

**Issue by woodiertexas** [QuiltMC/quilted-fabric-api#171]
Was messing around in a 1.21 dev environment for planetarium and came across this: https://mclo.gs/X1UNhbq

**Comment by AlexIIL** [comment]
Yeah so this is a weird one - I think the fix for this is https://github.com/QuiltMC/quilt-loader/pull/432, as I think remapping is the only reason that mods might get split up.

**Comment by woodiertexas** [comment]
oh ok

**Comment by MagmaSlime123** [comment]
Quilt simply refuses to load on 1.21.1.

**Comment by OroArmor** [comment]
Correct, please use 1.21 or loader overrides to get the 1.21 version to work on 1.21.1

**Comment by MagmaBro123** [comment]
Whatever that means, I do use 1.21. If the mod can't load on the version it requires, then please update it.

**Comment by MagmaSlime123** [comment]
> Correct, please use 1.21 or loader overrides to get the 1.21 version to work on 1.21.1

If the mod can't load on the version it requires, then please remove these unloadable mod versions.

**Comment by OroArmor** [comment]
They are not marked as compatible.
![Screenshot_20240917_211832_Edge.jpg](https://github.com/user-attachments/assets/9a1c4b62-8542-4fbf-abb2-7ae5cc283be3)
